pycode/memilio-generation/memilio/generation/graph_visualization.py

```diff
@@ -0,0 +1,191 @@
+#############################################################################
+# Copyright (C) 2020-2024 MEmilio
+#
+# Authors: Daniel Richter
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+"""
+@file graph_visualization.py
+@brief Class for plotting the abstract-syntax-tree in different formats. 
+"""
+import os
+from typing import TextIO
+from graphviz import Digraph
+from clang.cindex import Cursor
+from .ast import AST
+
+
+class Visualization:
+    """
+    Class for plotting the abstract-syntax-tree in different formats.
+    """
+    @staticmethod
+    def output_ast_terminal(ast: AST, cursor: Cursor) -> None:
+        """
+        Output the abstract syntax tree to terminal.
+
+        @param ast: ast object from AST class.
+        @param cursor: The current node of the AST as a cursor object from libclang.
+        """
+
+        _output_cursor_and_children_print(cursor, ast)
+
+        print(f'AST-terminal written.')
```

Comment:
I like the usage of the logging package, so i think this print should also be a logger info.